Classes/Systems/ShortcutHandler.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ShortcutHandler.swift
+//  Freetime
+//
+//  Created by Viktor Gardart on 2017-10-08.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+struct ShortcutHandler {
+
+    private struct Constants {
+        static let searchViewControllerIndex = 1
+    }
+
+    private enum Items: String {
+        case search
+        case switchAccount
+    }
+
+    static func configure(application: UIApplication, sessionManager: GithubSessionManager) {
+        application.shortcutItems = generateItems(sessionManager: sessionManager)
+    }
+
+    static func handle(shortcutItem item: UIApplicationShortcutItem,
+                       sessionManager: GithubSessionManager,
+                       navigationManager: RootNavigationManager) -> Bool {
+        let itemType = Items(rawValue: item.type)
+        switch itemType {
+        case .some(.search):
```

Comment:
Could we maybe change the `let itemType ...` to be a guard, and the switch reference the enum directly and not through the optional `.some` value?

Reply:
Sure!

Reply:
Done! 🙂 👍